doc/source/support/frequently-asked-questions.rst

```diff
@@ -134,6 +134,72 @@ Achieving the same result in Markdown often requires raw HTML, which is less rea
 
 Switching to ``README.rst`` at Level 5 helps users appreciate the formatting power of ``.rst`` and serves as a stepping stone toward writing full documentation in ``.rst`` as part of the ``scikit-package`` documentation standard.
 
+.. _faq-set-default-prompt-value:
+
+How can I change the default values that appear in the prompt?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To change the default values, create ``~/.skpkgrc``:
+
+.. code-block:: text
+
+  {
+      "default_context":
```

Comment:
too many examples. Again, we want to provide examples that are useful where people can simply change. "entry-name" and "new-default-value" don't provide much value and it's an extra step.